crates/wasi-threads/LICENSE

```diff
@@ -0,0 +1,220 @@
+
```

Comment:
If possible I think we're omitting LICENSE files except for the one at the top of the repo